src/components/screens/admin-vacation-management/EditVacationDialog.tsx

```diff
@@ -25,6 +30,19 @@ interface EditVacationDialogProps {
   disableSave: boolean;
 }
 
+/** 
+ * Generates a list of years: last year, current year and next year.
+ * @return An array of year strings.
```

Comment:
```suggestion
```

Reply:
I generally don't annotate the return unless there is a particular reason- it's quite clear what it is already